src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IReturnStatement.cs

```diff
@@ -421,6 +421,42 @@ int F(bool a)
             VerifyFlowGraphForTest<BlockSyntax>(compilation, expectedGraph);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void ReturnFlow_UnaryNegation()
```

Comment:
>ReturnFlow_UnaryNegation [](start = 20, length = 24)

I think this test really belongs in IOperationTests_IUnaryOperatorExpression.cs. It looks like we don't have a test covering ```!!f``` scenario, please add one, I expect to not see a IUnaryOperation in the flow graph for it. Also, please add similar tests for VB.

Reply:
c96b2be60a45fe1f73df97f27ebaedc7d635f7c2 9d525ebc29b3ff5848c87645e2ed412f4e65c3a9